src/main/webapp/assets/createfile.css

```diff
@@ -1,4 +1,7 @@
 body {
+  background-image: linear-gradient(to right,rgb(220, 220, 220),rgb(169, 169, 169));
```

Comment:
since these are shared across all of your CSS files, you should be able to put them in a body {} selector in general.css, as long as that stylesheet is loaded across all of our pages

Reply:
Looks like maybe "body" is identical across all pages, so it's probably better to define it once